test-data/unit/check-newtype.test

```diff
@@ -277,8 +277,8 @@ A = NewType('A', T)
 B = NewType('B', List[T])
 [builtins fixtures/list.pyi]
 [out]
-main:3: error: Invalid type "__main__.T"
 main:3: error: Argument 2 to NewType(...) must be subclassable (got T?)
+main:3: error: Invalid type "__main__.T"
```

Comment:
@gvanrossum The order of these two messages is changed because now the column is correctly passed and errors are ordered by column.